### PR TITLE
Day 1 -> Project

### PR DESCRIPTION
**Pull request by anthonypiazza**

### CODE REVIEW

File: instagram-app/src/components/SearchBar/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className="search">
+      <h1>Instagram </h1>
+      <input
+        type="text"
+        placeholder="Search"
+      />
+      <i className="far fa-compass"></i>
+      <i className="far fa-heart"></i>
+      <i className="fas fa-user-alt"></i>
```
Comment (sebaghdev) [inline instagram-app/src/components/SearchBar/SearchBar.js:14]: Increase icon sizes

File: instagram-app/src/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+function SearchBar() {
+  return (
+    <div className="search">
+      <h1>Instagram </h1>
```
Comment (sebaghdev) [inline instagram-app/src/components/SearchBar/SearchBar.js:7]: use instagram logo

File: instagram-app/src/components/Post/Post.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <div className="post-header">
+            <img className="user-photo" src={props.thumbnail} alt={props.username}/>
+            <h2>{props.username}</h2>
+        </div>
+        <div className="post-image">
+            <img src={props.image} alt={props.username}/>
+        </div>
+        <div className="icons">
+          <i className="far fa-heart"></i>
+          <i className="far fa-comment"></i>
```
Comment (sebaghdev) [inline instagram-app/src/components/Post/Post.js:16]: Increase icon size and fix css.

File: instagram-app/.gitignore [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (sebaghdev) [inline instagram-app/.gitignore:1]: Remove serviceworkers/logo.svg

Not needed.

File: instagram-app/src/components/PostContainer/PostContainer.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import '../Post/Post.css'
+
+const PostContainer = props => {
+  return (
+    <div className="post-container">
+      {props.posts.map((post) => {
+        return(
+          <div>
+            <Post likes={post.likes} image={post.imageUrl} thumbnail={post.thumbnailUrl} username={post.username} key={post.timestamp} />
+            <CommentSection comments={post.comments} key={post.id} />
```
Comment (sebaghdev) [inline instagram-app/src/components/PostContainer/PostContainer.js:14]: You can format this differently to make it more readable like so:

            <Post
              likes={post.likes}
              image={post.imageUrl}
              thumbnail={post.thumbnailUrl}
              username={post.username}
              key={post.timestamp}
            />

            <CommentSection
              comments={post.comments}
              key={post.id}
            />

**Comment by anthonypiazza** [comment]
Thanks for the input! I went ahead and pushed the edits to my Day 2 branch as my current files include all of today's work as well.

**Pull request merged**